include/libtorrent/alert_types.hpp

```diff
@@ -1815,8 +1815,8 @@ namespace libtorrent
 	struct TORRENT_EXPORT add_torrent_alert final : torrent_alert
 	{
 		// internal
-		add_torrent_alert(aux::stack_allocator& alloc, torrent_handle h
-			, add_torrent_params const& p, error_code ec);
+		add_torrent_alert(aux::stack_allocator& alloc, torrent_handle const& h
```

Comment:
right, my comment only applied to ``torrent_handle``, which basically is a ``std::weak_ptr``. And this particular case may be a contrived example, but since we're taking a copy of this anyway, if we expose the fact that we want a copy (that we can then move into place), the caller has the ability to do the same, and redundant copies can be avoided in favor of moves. anyway, no need to change this